tests/application_tests/src/test_report.py

```diff
@@ -47,7 +47,7 @@ def setUp(self):
         self.driver = webdriver.Remote("http://selenium:4444", options=firefox_options)
         self.driver.implicitly_wait(10)
         self.wait = WebDriverWait(self.driver, 10)
-        self.driver.get("http://www:80")
+        self.driver.get("http://www:80?hide_toasts=true")
```

Comment:
Do we also test case false?

Reply:
Added a comment to explain:

```suggestion
        self.driver.get("http://www:80?hide_toasts=true")  # Hide toasts so they don't block other visual elements
```